### ISSUE TITLE
Nickname commands have wrong capture groups if alias_nicks set

### ISSUE THREAD

**Issue by dgw** [sopel-irc/sopel#1343]
**tl;dr:** If you run Sopel from source, apply the two-character patch from cf6acce to your local tree for correct nickname command function. The bug only exists on the `master` branch.

----

After merging #1052, I discovered a problem with nickname commands. Reloading modules, for example, using the updated `reload.py` from #1053 was reloading `module 'reload'`, always.

The added regex capture group containing the bot's nick and configured aliases means nickname commands get an extra group compared to regular commands, **only if `core.alias_nicks` is set:**

https://github.com/sopel-irc/sopel/blob/a9d04a4899f6a1d8046d11d1b442f9b9e6e73f61/sopel/loader.py#L118-L124

It's easy to fix—just make that group non-capturing and Bob'll be my uncle—but it troubles me that it seems Sopel's test suite cannot currently catch this bug. Even though I missed the addition of a new capture group, the tests should have been able to catch it.
